modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -226,7 +227,7 @@ const makeNoopAdaptor = (): RtcAdaptor => {
     formatter: {
       match: Fun.never,
       matchAll: Fun.constant([]),
-      matchNode: Fun.never,
+      matchNode: Fun.constant(undefined),
```

Comment:
Since we are currently throwing on the RTC side for this function call and this noop interface is only used when the editor is in the initializing state the impact should be low. But good to get the right type here.